class/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(id, age, specialization, name: 'Unknown', parent_permission: true)
+    super(id, age, name, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Hello! 📌 Please remember that there's no need to inherit or pass the id as a parameter to the subclass since it's automatically generated. 🆔🔧 This simplifies the inheritance process and ensures that the unique identifier is managed seamlessly. Furthermore, in accordance with the project guidelines, it's important to emphasize that the specialization parameter should not default to "unknown." 🚫❓ This helps maintain consistency with the project's specific requirements and ensures that classroom information is provided explicitly when necessary. 🏫📄

```suggestion
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```